Source/CameraView/CameraMan.swift

```diff
@@ -19,6 +20,10 @@ class CameraMan {
   var stillImageOutput: AVCaptureStillImageOutput?
   var startOnFrontCamera: Bool = false
 
+  init(withConfiguration configuration: Configuration) {
```

Comment:
The Swift convention is to just use variable names as parameter labels in initializers. So here the `withConfiguration` can be dropped.